### DISCUSSION TITLE
Get model field by name

### DISCUSSION THREAD

**Discussion by Pirulax** [tortoise/tortoise-orm#1692] [Q&A]
I need it to find a field by it's name, since `Model.id` (let's assume `id = IntField(....)`) gives me an `AttributeError`.
I did find a way involving `_meta`, but since it's seemingly a private field (has `_` prefix), I'd like to find a more "official" solution.

For context, I made something like this for my Sanic app:
```py
def inject_object_from_url(orm_type : Type[tortoise.Model], key_field_name : str = 'id', arg_name : str | None = None):
    key_field = orm_type._meta.fields_map[key_field_name] # - If `KeyError`: Make sure the field exists in the model # pylint: disable=0212
    key_type : Type = key_field.field_type
    if key_type not in (str, int):
        raise ValueError(f'Field must either by of type `str` or `int`, got {key_type.__class__.__name__}')

    if not arg_name:
        arg_name = orm_type.__name__.lower()
    key_field_name = key_field.model_field_name
    param_name = f'{arg_name}_{key_field_name}'
    
    #...
```

Thank you!

**Comment by henadzit** [comment]
There is [Model.describe](https://tortoise.github.io/setup.html?h=describe#tortoise.Model.delete-return-type) that returns a dictionary containing the model description.
